tests/functional/cylc-combination-scripts/10-CYLC_WORKFLOW_SRC_DIR.t

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC WORKFLOW ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#------------------------------------------------------------------------------
+# Test `cylc vip` (Validate Install Play)
+
+. "$(dirname "$0")/test_header"
+
+set_test_number 4
+
+WORKFLOW_NAME="cylctb-x$(< /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c6)"
+
+cat > flow.cylc <<__HERE__
+#!jinja2
+# TEST: {{ CYLC_WORKFLOW_SRC_DIR }}
+[scheduler]
+    allow implicit tasks = true
+[scheduling]
+    [[graph]]
+        R1 = foo
+__HERE__
+
+# It starts playing:
+run_ok "${TEST_NAME_BASE}-vip" \
+    cylc vip \
```

Comment:
Would be good to add a `cylc view -p` test here (works when I try it manually) to make sure this variable is provided in that context too (note config parsing doesn't happen here so it's a likely breakage).